pubsub/google/cloud/pubsub/_gax.py

```diff
@@ -148,7 +148,7 @@ def topic_delete(self, topic_path):
                 raise NotFound(topic_path)
             raise
 
-    def topic_publish(self, topic_path, messages):
+    def topic_publish(self, topic_path, messages, timeout=30):
```

Comment:
What is the rationale behind 30 seconds as the default? (Not saying it is wrong necessarily, just wondering why that choice.)